README.md

```diff
@@ -41,18 +41,21 @@ Available inputs for `monitor`:
 The behaviour for monitoring of directories for sequencing runs to upload is controlled through the use of a JSON config file. An example may be found [here](https://github.com/eastgenomics/s3_upload/blob/main/example/example_config.json).
 
 The top level keys that may be defined include:
-* `max_cores` (int): maximum number of CPU cores to split uploading across (default is the maximum available)
-* `max_threads` (int): the maximum number of threads to use per CPU core
-* `log_level` (str): the level of logging to set, available options are defined [here](https://docs.python.org/3/library/logging.html#logging-levels)
-* `log_dir` (str): path to where to store logs (default is `/var/log/s3_upload`)
+* `max_cores` (int | optional): maximum number of CPU cores to split uploading across (default: maximum available)
+* `max_threads` (int | optional): the maximum number of threads to use per CPU core
+* `log_level` (str | optional): the level of logging to set, available options are defined [here](https://docs.python.org/3/library/logging.html#logging-levels)
+* `log_dir` (str | optional): path to where to store logs (default: `/var/log/s3_upload`)
+* `slack_log_webhook` (str | optional): Slack webhook URL to use for sending notifications on successful uploads, will try use `slack_alert_webhook` if not specified.
```

Comment:
Why does it use 'slack_alert_webhook' if slack_log_webhook is not specified? And vice versa?